shells/webextension/popups/development.html

```diff
@@ -2,7 +2,7 @@
 <style>
   html, body {
     font-size: 14px;
-    min-width: 430px;
+    min-width: 460px;
```

Comment:
This is unrelated but Chrome probably changed default fonts in popups because they started wrapping onto the next line a few versions ago. I made all popups 30px wider to make them look nice again.

Reply:
Thanks for the explanation. 👍